resources/RemoveInactiveUsersRunner.php

```diff
@@ -57,36 +72,58 @@
     if ($lastLoginDate) { // null lastLoginDate check
         $interval = $today->diff($lastLoginDate);
     } else { // This might only be run once, since new users always have field filled.
-        echo "User has no lastLoginDate (it may have been a very long time.)\n";
-        echo "Deleting user.\n";
+        echo "Deleting this user as it has no last login date " .
+        "(it may have been a very long time). \n";
+        //writing to file to say who would have been deleted
+        file_put_contents("InactiveUsersToBeDeleted.txt", "The user " .
+        "with Id ".$user->getId()." would have been deleted as it " .
+        "has no last login date.\n", FILE_APPEND);
```

Comment:
Similar to the other delete later on, we only want to log to a file in dry run mode - so an if statement is needed here.

Reply:
According to an earlier comment left before I edited, there shouldn't be any users without a last login date, as all new users will have one, and any users without a last login date would have been deleted the first time the program was run (which has already happened? not sure).  But ok will fix.